includes/definitions/discovery/fxm.yaml

```diff
@@ -32,16 +34,17 @@ modules:
                     oid: upsOutputFrequency
                     num_oid: .1.3.6.1.4.1.7309.6.1.4.2.
                     index: 'upsOutputFrequency.{{ $index }}'
-                    divisor: 10
                     descr: Output
+                    divisor: 10
         current:
+            options:
+              divisor: 10
```

Comment:
This doesn't change the divisor. You've just moved it to be applied to all currents detected.

Reply:
Ignore that